waspc/e2e-test/ShellCommands.hs

```diff
@@ -83,6 +88,14 @@ insertCodeIntoFileAtLineNumber fileName atLineNumber line =
         "mv " ++ fileName ++ ".tmp " ++ fileName
       ]
 
+replaceLineInFile :: FilePath -> Int -> String -> ShellCommandBuilder ShellCommand
```

Comment:
Ha these e2e tests might be a good place to use .diff s! Just an idea for the future.